framework/src/utils/ParallelUniqueId.C

```diff
@@ -18,7 +18,7 @@ bool ParallelUniqueId::_initialized = false;
 
 #ifdef LIBMESH_HAVE_TBB_API
 tbb::concurrent_bounded_queue<unsigned int> ParallelUniqueId::_ids;
-#elif LIBMESH_HAVE_PTHREAD
+#elif !defined(LIBMESH_HAVE_OPENMP) && defined(LIBMESH_HAVE_PTHREAD)
```

Comment:
This will be gone when reading. It is a commit from a PR that was merged. See the original PR description.